cmake/SetupLapack.cmake

```diff
@@ -0,0 +1,10 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+find_package(LAPACK REQUIRED)
+message(STATUS "LAPACK_LIBRARIES:  " ${LAPACK_LIBRARIES})
```

Comment:
`LAPACK libs: ${LAPACK_LIBRARIES}` to be consistent with all the other libs. The BLAS one should be changed too since this follows that one